### ISSUE TITLE
Volume slider for music doesn't work [iOS]

### ISSUE THREAD

**Issue by simmel** [abergs/listentotheclouds#2]
At least it isn't possible to silence the music by sliding it down to "zero". I suspect it doesn't control the sound level at all.
All this on iOS.

**Comment by abergs** [comment]
Yeah, unfortunately iOS doesn't allow to control the volume on HTML5 audio :(

**Comment by simmel** [comment]
= / Then I guess you can close this.

Maybe note it somewhere?
